tests/h/views/client_test.py

```diff
@@ -77,6 +75,12 @@ def pyramid_settings(pyramid_settings):
     return pyramid_settings
 
 
+@pytest.fixture
+def pyramid_request(pyramid_request):
+    pyramid_request.feature.flags["embed_cachebuster"] = False
+    return pyramid_request
```

Comment:
This feature flag is a development aid to bypass the caching of https://cdn.hypothes.is/hypothesis in production by adding a cache-busting query param whenever that URL is referenced. Disabling it is a more appropriate default for these tests.